parlai/crowdsourcing/tasks/dialcrowd/analysis/compile_results.py

```diff
@@ -0,0 +1,543 @@
+#!/usr/bin/env python3
```

Comment:
Yeah, files like this seem to be largely copied from current code. Can these be removed from the PR or are they used in some way? If they're being used, it'd be good to remove the duplication with the existing code